### PR TITLE
Added kick command.

### PR DESCRIPTION
**Pull request by spacechase0**
I assume this is the right way to do this (both the pull request and the command)? :P

### CODE REVIEW

File: src/main/java/org/spout/vanilla/command/AdministrationCommands.java [hunk 1]
```diff
@@ -111,6 +111,19 @@ public void gamemode(CommandContext args, CommandSource source) throws CommandEx
 			source.sendMessage(player.getName() + "'s game mode has been changed to " + message);
 		}
 	}
+	
+	@Command( aliases = "kick", usage = "[player]", desc = "Kick a player from the server", max = 1 )
```
Comment (windy1) [inline src/main/java/org/spout/vanilla/command/AdministrationCommands.java:115]: We don't use spaces in between parentheses like that it should be, <code>@Command(aliases = "kick" ...)</code>
Same goes for the rest of the pull (for the future)

**Comment by windy1** [comment]
Kick is a Spout command and has already been implemented.

**Comment by Zidane** [comment]
@WalkerCrouse speaks the truth.

**Comment by spacechase0** [comment]
Aww. Guess I'll look for something else. :P Trying to find something small, since I don't know much of the codebase.

**Comment by Zidane** [comment]
Don't feel put off @spacechase0, this bit of code just doesn't fit in Vanilla. If you get another idea for an improvement...don't hesitate to put up another PR!

**Comment by Darsstar** [comment]
And just like you can set the maximum allowed parameters with `max` you can set the minimum required parameters with `min`.
Then you don't need that `if (args.length() == 1)`

**Pull request closed without merging**